src/pybind/mgr/volumes/fs/subvolume.py

```diff
@@ -4,38 +4,18 @@
 LGPL2.1.  See file COPYING.
 """
 
-import errno
 import logging
 import os
+import errno
 
 import cephfs
-import rados
 
+from subvolspec import SubvolumeSpec
```

Comment:
s/subvolspec/.subvolspec/

Reply:
Without this change I hit the error that the subvolspec module couldn't be found